.yamllint.yml

```diff
@@ -5,7 +5,7 @@ rules:
   empty-values:
     forbid-in-block-mappings: true
   line-length:
-    max: 79
+    max: 100
```

Comment:
This will cause an overrun on my VT100!

Reply:
ok I can change it back :) - I'd really love to be able to run the linter and the unit tests on this repo.  Some of our descriptions are long enough that I was getting linting errors for line length (and linting errors about trailing spaces when I tried to break the description up into multiple lines) which felt a bit too constrictive.  But I don't feel strongly. :) 

It would be really nice to be able to run tests in this repo too, we have an open issue about that with reasoner-validator team.  https://github.com/NCATSTranslator/reasoner-validator/issues/56